tsconfig.json

```
@@ -0,0 +1,14 @@
+{
+  "compilerOptions": {
+    "module": "esnext",
```

We should avoid moving pointers and use a fixed version to output.

```suggestion
    "module": "es2020",
```